services/repository/src/test/java/org/sagebionetworks/repo/web/service/EvaluationServiceTest.java

```diff
@@ -102,7 +102,7 @@ public void testGetAllSubmissions() {
 		List<Submission> expectedRes = new LinkedList<Submission>();
 		when(mockSubmissionManager.getAllSubmissions(userInfo, evalId,  SubmissionStatusEnum.OPEN, limit, offset)).thenReturn(expectedRes);
 		// Call under test
-		evaluationService.getAllSubmissions(userId, evalId, SubmissionStatusEnum.OPEN, limit, offset, null);
+		evaluationService.getAllSubmissions(userId, evalId, SubmissionStatusEnum.OPEN, limit, offset);
```

Comment:
please backfill basic test for getEvaluationsInRange and getEvaluationByContentSource